src/structurizr/model/groupable_element.py

```diff
@@ -0,0 +1,56 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Provide a superclass for all elements that can be included in a group."""
+
+
+from abc import ABC
+from typing import Optional
+
+from .element import Element, ElementIO
+
+
+__all__ = ("GroupableElementIO", "GroupableElement")
+
+
+class GroupableElementIO(ElementIO, ABC):
+    """
+    Define a superclass for all elements that can be included in a group.
+
+    Attributes:
+        group (str): The name of thegroup in which this element should be included.
+    """
+
+    group: Optional[str] = ""
```

Comment:
`Optional` denotes that `group` can be `None` but you set the empty string as the default value. Is `None` allowed? If it is always at least an empty string, please remove the `Optional`.

Reply:
Good catch.  I've made it mirror the Java implementation, where no group is indicated by `null` rather than empty string.